### PR TITLE
Update WPILib

### PR DESCRIPTION
**Pull request by TheMageKing**
The WPILib in this repository is years out of date from the actual WPILib.  Additonally, it appears to only include the documentation for the Java version of the library, not the C++ version.  I added both the C++ and the updated java versions.  Can you merge it?

### CODE REVIEW

File: docsets/WPILib_Cpp/docset.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "WPILib_Cpp",
```
Comment (Kapeli) [inline docsets/WPILib_Cpp/docset.json]: Please set the `name` to "WPILib C++"

File: docsets/WPILib_Cpp/docset.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "version": "2020.2.2",
+    "archive": "WPILib_Cpp.tgz",
```
Comment (Kapeli) [inline docsets/WPILib_Cpp/docset.json]: The archive at this path contains the WPILib Java docset, not the C++ one

File: docsets/WPILib_Java/docset.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "WPILib_Java",
```
Comment (Kapeli) [inline docsets/WPILib_Java/docset.json]: Please set the `name` to "WPILib Java"

File: docsets/WPILib_Java/docset.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    "version": "2020.2.2",
+    "archive": "WPILib_Java.tgz",
```
Comment (Kapeli) [inline docsets/WPILib_Java/docset.json]: If possible, please set the docset's name (the `CFBundleName` within `Info.plist`) to "WPILib Java"

File: docsets/WPILib_Cpp/docset.json (hunk 2 above)
Comment (TheMageKing) [reply docsets/WPILib_Cpp/docset.json]: Whoops.  Guess I messed up one of my moves...
Good catch: thanks

File: docsets/WPILib_Java/docset.json (hunk 4 above)
Comment (TheMageKing) [reply docsets/WPILib_Java/docset.json]: I'll do the same for the C++ version as well (once I actually include it lol)

Comment (TheMageKing) [reply docsets/WPILib_Java/docset.json]: Should I touch the other keys as well?  Eg, DocSetFeedName or Publisher?

Comment (Kapeli) [reply docsets/WPILib_Java/docset.json]: Those keys aren't used by Dash. `DocSetPlatformFamily` and `CFBundleName` are the important ones for Dash.

**Comment by TheMageKing** [comment]
Doxygen appears to have failed to generate an index for the C++ files... what?
I did it according to the websites directions...
Should DISABLE_INDEX be false?

**Comment by TheMageKing** [comment]
Okay, so, it seems that I need docsetutil to generate an index for my docset... but I am on a Linux box, and just don't have access to that.
What do you recommend?

**Comment by Kapeli** [comment]
I'm sorry, but I have no idea what you should do in this case. I only handle the Mac side of things.

**Comment by Kapeli** [comment]
I'm sorry, but no. I can't commit to doing this every time a new version is released.

**Pull request closed without merging**